File: week1.R
# R course for beginners
# week 1

subject_number = c(1:6)
gender = c("F","M","F","M","M","F")
age = c(27,24,30,26,40,35)
depression = c(0,0,1,0,1,0)

df = data.frame(subject_number, gender, age, depression)

write.csv(df, file = "C:/Users/danie/OneDrive/Desktop/code/subjects.csv")

File: week1.py
# Python course for beginners
# week 7

import pandas as pd

data = {
    "subject_number": np.arange(1,101,1),
    "age": np.random.randint(15,46,size=100),
    "gender": np.random.choice(['F','M'], size=100),
    "depression": np.random.binomial(n=1, p=0.15, size=100)
}

df = pd.DataFrame(data)
df.to_csv("subjects.csv", index=False)